multi-data-centers-in-one-city-deployment.md

```diff
@@ -71,6 +71,10 @@ member leader_priority pdName2 4
 member leader_priority pdName3 3
 ```
 
+> **Note:**
+> 
+> Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [Placement Rules](/configure-placement-rules.md).
```

Comment:
```suggestion
> Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [placement rules](/configure-placement-rules.md).
```